object.js

```diff
@@ -0,0 +1,41 @@
+const favMovie = {
+  title: "Howl's Moving Castel",
+  year: 2004,
+  director: "Hayao Miyazaki",
+  actors: "Sofie",
+  rating: 8.2,
+  timesWatched: 100,
+  recommend: function () {
+    console.log(`you have to watch ${this.title}`);
+  },
+  watch: function () {
+    console.log(this.timesWatched + 1);
+  },
+  setNewRating: function (newRating) {
+    this.rating = newRating;
+    console.log(this.rating);
+  },
+  addActor: function (actorName) {
+    this.actors = `${this.actors} , ${actorName}`;
```

Comment:
This is supposed to add a new actor each time, but what you did will always replace the last added one. You can use the plus sign:
`this.actors = this.actors + ", " + actorName;`
or use the concat method:
`this.actors = this.actors.concat(", ", actorName);`